launch.bat

```diff
@@ -66,10 +68,22 @@ goto Start
     :: Check for python installation
     where "%PYTHON_CMD%" >nul 2>&1
     if %errorlevel% neq 0 (
-        echo Python is not found.
+        echo %PYTHON_CMD% is not found.
+
+        if "%TEST_OTHER_PYTHON_PATH%"=="true" (
+            echo Try with py
```

Comment:
```suggestion
            echo Checking if python is available as py...
```
nit: I like to use action-`ing` and an ellipse for logs where something is about to happen.